modules/caddyhttp/fileserver/matcher_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2015 Matthew Holt and The Caddy Authors
+// Copyright 2016 Matthew Holt and The Caddy Authors
```

Comment:
Any reason for this?

Reply:
i have no clue how this happened lol. maybe i accidentally pressed the increment hotkey in vim... ill change it back so there's no diff